users/dierkes/preprocessing/segment_cutting.py

```diff
@@ -0,0 +1,85 @@
+import soundfile as sf
+import numpy as np
+from i6_core.lib import corpus
+import multiprocessing
+import tqdm
+
+from sisyphus import Job, Task, tk
+
+class CutSpeechSegmentsFromAudio(Job):
```

Comment:
Maybe the name could reflect a bit better what the job does and it should end on `"Job"`, see https://github.com/rwth-i6/i6_core/blob/main/CONTRIBUTING.md

Maybe `CutAndStitchSpeechSegmentsFromCorpusJob`?